application-packages/Stacker/DeployProcess_Stacker.json

```diff
@@ -23,8 +23,8 @@
                             "default": true
                         }
                     ],
-                    "minOccurs": 1,
-                    "maxOccurs": "unbounded",
+                    "minOccurs": "1",
+                    "maxOccurs": "2",
```

Comment:
It is nice that the EMS is now able to limit the number of OpenSearch result based on this param, but we still need to advertise our application as an unbounded one.

Reply:
fixed